components/uid.rst

```diff
@@ -370,6 +370,121 @@ of the ULID parameters::
         }
     }
 
+Generating and Inspecting UUIDs/ULIDs in the Console
+----------------------------------------------------
+
+This component provides several commands to generate and inspect UUIDs/ULIDs in
+the console. They are not enabled by default, so you must add the following
+configuration in your application before using these commands:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # services.yaml
+        services:
+            Symfony\Component\Uid\Command\GenerateUlidCommand: ~
+            Symfony\Component\Uid\Command\GenerateUuidCommand: ~
+            Symfony\Component\Uid\Command\InspectUlidCommand: ~
+            Symfony\Component\Uid\Command\InspectUuidCommand: ~
```

Comment:
I understand why they are optional, but this seems like a really non-DX friendly feature this way. @fancyweb @nicolas-grekas is there a particular reason not to add a config option for this?

Reply:
can those commands be enabled by default but configured as https://symfony.com/doc/current/console/hide_commands.html instead?